pages.es/common/acyclic.md

```diff
@@ -0,0 +1,17 @@
+# acyclic
+
+> Hace un gráfico acíclico invirtiendo algunos bordes.
+> Filtros Graphviz: `acyclic`, `bcomps`, `comps`, `edgepaint`, `gvcolor`, `gvpack`, `mingle`, `nop`, `sccmap`, `tred`, & `unflatten`.
+> Más información: <https://graphviz.org/pdf/acyclic.1.pdf>.
+
+- Hace un gráfico acíclico invirtiendo algunos bordes:
+
+`acyclic {{ruta/a/entrada.gv}} > {{ruta/a/salida.gv}}`
```

Comment:
```suggestion
`acyclic {{ruta/al/entrada.gv}} > {{ruta/al/salida.gv}}`
```
Shouldn't it be `al` instead of `a`.

Reply:
Not, because it's another gender. M `al` vs F `a la`. Leave as-is.